drupal-org.make

```diff
@@ -433,7 +436,7 @@ projects[total_control][version] = 2.x-dev
 ;;;;;;;;;;;;;;;;;;;;;
 
 projects[varbase_editor][type] = module
-projects[varbase_editor][version] = 4.x-dev
+projects[varbase_editor][version] = 4.17
```

Comment:
Please do not change the version of Vardot's developed modules. as we are in 
8.x-dev mode .. we will keep pushing work to each public project.
And just before we release varbase 8.4.07 we will do a mass release for all projects.

So for this time we will use varbase_editor 4.x-dev .. and entityqueue_form_widget 1.x-dev .

Any part of the team could push a change/fix to his maintained project, and use the x-dev for testing with varbase 8.4.x-dev 

composer create-project vardot/varbase-project:8.4.x-dev PROJECT_DIR_NAME --stability dev --no-interaction